shared/src/main/scala/org/adridadou/openlaw/parser/template/variableTypes/PeriodType.scala

```diff
@@ -39,6 +39,10 @@ case object PeriodType extends VariableType("Period") {
     }
   }
 
+  def castToString(value:String, executionResult:TemplateExecutionResult): String = {
+    value
```

Comment:
I think the best way to handle that is to change the "internal value" to become the value we want to print because we already have the parser ready for it